src/convertToObject.js

```diff
@@ -7,6 +7,29 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const result = {};
+  const regex = /\n+|\t+/g;
+
+  sourceString
+    .replaceAll(regex, ' ')
```

Comment:
replaceAll doesn't exists in 14 node version.
it is a reason of failed tests and in general redundant here.
![Screenshot 2024-08-25 at 17 50 03](https://github.com/user-attachments/assets/404bccdd-e509-410e-9dd8-689d79ba8b4f)